main.cpp

```diff
@@ -537,6 +554,15 @@ double Equalizer::adjustWidth(double level)
     return level;
 }
 
+int Equalizer::addBand()
+{
+    while(numBands < 8)
+    {  
+        return numBands + 1;        
+    }
+    return numBands;
```

Comment:
this only loops once, because the first line in your `while()` loop is a `return` statement. 
- [ ] revise so it actually loops multiple times before returning.